README.md

```diff
@@ -10,6 +10,16 @@ This recieves the payload from the [`analytics` integration](https://www.home-as
 
 The worker source are located in the worker directory.
 
+### Schedule
+
+As a part of the worker, there is several scheduled tasks that iterates over the stored data and makes a generated object that the site uses, these tasks run at different times.
```

Comment:
```suggestion
There are several scheduled tasks that iterate over the stored data and generate the data that the site uses. These tasks run at different times.
```

Reply:
That should not be committed her (directly to dev)

Reply:
https://github.com/home-assistant/analytics.home-assistant.io/pull/129 @balloob